src/configs/localeConfig.js

```diff
@@ -38,6 +43,7 @@ export default {
   // pairs of language code and language name in native language
   supportedLanguages: {
     [DEFAULT_LANGUAGE_CODE]: DEFAULT_LANGUAGE,
+    am: 'Amharic',
```

Comment:
Value (language name) should be in native language (for better UX) :)

Reply:
Thanks for pointing that out - I'll get this changed 👍🏾

Reply:
All done ✅